doc/DEVELOPERS.rst

```diff
@@ -50,25 +50,11 @@ activate a venv::
 
     git clone https://github.com/rst2pdf/rst2pdf
     cd rst2pdf
-    python3 -m venv .venv
-    . .venv/bin/activate
+    uv sync --all-extras
```

Comment:
As above, this shows the power of `uv` compared to the old setup process on the LHS.

Reply:
I think `uv` is growing on me :)